src/modules/presets.js

```diff
@@ -0,0 +1,64 @@
+/**
+  @function configPrefix
+  return an piece of settings object, configured by a 'prefix' or 'no-prefix' presets
+  @param include {boolean} - the value of the object properties for message prefix configuration
+  @return {object} - a piece of settings, responsible for prefix configuration
+**/
+function configPrefix(include) {
+  return {
+    LOG_TIMESTAMP: include,
+    LOG_ENVIRONMENT: include,
+    LOG_LEVEL: include,
+    LOG_REQID: include
+  };
+}
+
+/**
+  @function applyPreset
+  @param preset {string} - a key name of a preset
+  @return {object} - a piece of settings object based on the preset value
+**/
+function applyPreset(preset) {
```

Comment:
## Suggestion
It can be refactored to use `return` statement instead of break, which will avoid using the `else` and avoid using the `result` object.